templates/cli/lib/commands/push.js.twig

```diff
@@ -844,10 +844,14 @@ const attributesToCreate = async (remoteAttributes, localAttributes, collection,
 
     if (!cliConfig.force) {
         if (deleting.length > 0 && !isIndex) {
-            log(`Attribute deletion will cause ${chalk.red('loss of data')}`);
+            console.log(`${chalk.red('-------------------------------------------------------')}`);
+            console.log(`${chalk.red('| WARNING: Attribute deletion will cause loss of data |')}`);
```

Comment:
I think we should replace `will` with `may`.
`Will` sounds too strict, too powerful. For example when collection dont have any documents yet, the warning isnt relevant